lib/aruba/api/deprecated.rb

```diff
@@ -1017,60 +1000,6 @@ def stderr_from(cmd)
         process_monitor.stderr_from(cmd)
       end
 
-      # @deprecated
-      #
-      # Get stdout of all processes
-      #
-      # @return [String]
-      #   The stdout of all process which have run before
-      def all_stdout
-        Aruba.platform.deprecated(
-          'The use of "#all_stdout" is deprecated.' \
-          ' Use `all_commands.map { |c| c.stdout }.join("\n") instead.' \
-          ' If you need to check for some output, use' \
-          ' "expect(all_commands).to included_an_object have_output_on_stdout(/output/)"' \
-          ' instead'
-        )
-
-        process_monitor.all_stdout
```

Comment:
these delegate to process_monitor but they have been reintroduced in commands.rb as a command_monitor.

I'm assuming this is correct.

Reply:
Yes, that's correct. `process_monitor` is also deprecated.